src/react/deckgl.js

```diff
@@ -126,6 +128,18 @@ export default class DeckGL extends React.Component {
       );
     }
 
+    // TODO: add handlers on demand at runtime, not all at once on init
+    this.eventManager = createEventManager(canvas, {})
+    .on({
+      click: this._onClick,
```

Comment:
Yeah... app will want to when the hover target change and / or when data changed, etc... but is probably don't want to know when camera / viewport should be changed. 

This is just a discussion, not a request to change the code. I was just wondering if it's reasonable to provide the callback API at higher level... say instead of giving them onClick(), we should give them onItemsPicked() and onItemsPicked() will get called when the user tries to pick items in various ways, clicking, dragging, keyboard selection, etc..